remote-api/src/main/java/org/alfresco/rest/api/model/rules/SimpleCondition.java

```diff
@@ -91,92 +90,20 @@ public static List<SimpleCondition> listOf(final List<ActionCondition> actionCon
      * @param actionCondition - {@link ActionCondition} service POJO
      * @return {@link SimpleCondition} REST model
      */
-    public static SimpleCondition from(final ActionCondition actionCondition, final NamespaceService namespaceService)
+    public static SimpleCondition from(final ActionCondition actionCondition,
+                                       final RestModelMapper<SimpleCondition, ActionCondition> simpleConditionMapper)
     {
-        if (actionCondition == null || actionCondition.getActionConditionDefinitionName() == null || actionCondition.getParameterValues() == null)
+        if (actionCondition == null || actionCondition.getActionConditionDefinitionName() == null ||
+                actionCondition.getParameterValues() == null || actionCondition.getParameterValues().isEmpty())
```

Comment:
Why did you add `actionCondition.getParameterValues().isEmpty()` to this condition?

Reply:
Thanks for spotting that - actually I think I meant to replace `actionCondition.getParameterValues() == null` with `actionCondition.getParameterValues().isEmpty()` as it appears that at the moment `actionCondition.getParameterValues()` can never be null:
```
public Map<String, Serializable> getParameterValues()
    {
        Map<String, Serializable> result = this.parameterValues;
        if (result == null)
        {
            result = new HashMap<String, Serializable>();
        }
        return result;
    }
```
Would that make sense?

Reply:
It's not clear to me what this should say. I just checked the `toRestModel` method - and it starts with:
```
if (actionCondition == null || actionCondition.getActionConditionDefinitionName() == null || actionCondition.getParameterValues() == null)
        {
            return null;
        }
```

So maybe we can remove this validation completely and use the check in `toRestModel` instead?

Reply:
I meant to get rid of `SimpleCondition` duplicate part when removing `SimpleCondition::from` but you're right - this can be done within this PR. 
I'll push changes soon.